server/s3_iem.h

```diff
@@ -110,4 +110,10 @@ extern S3Option* g_option_instance;
   "Metadata has accessibility issues. Contact Seagate Support."
 #define S3_IEM_METADATA_CORRUPTED_JSON ""
 
+#define S3_IEM_CHECKSUM_MISMATCH "0030060004"
```

Comment:
[* suggestion *] Add bucket, object name, Motr object id to the IEM message.

Reply:
Also expected and calculated checksum.

Reply:
Extended with bucket name, object name, motr id, md5 calc, md5 read